pyvista/core/filters/poly_data.py

```diff
@@ -1593,28 +1597,32 @@ def decimate(
             volume preservation is disabled and if ``attribute_error``
             is active, these errors can be large.
 
-        attribute_error : bool, default: False
-            Decide whether to include data attributes in the error metric. If
-            ``False``, then only geometric error is used to control the
-            decimation. If ``True``, the following flags are used to specify
-            which attributes are to be included in the error calculation.
+        boundary_constraints: bool, default: False
+            Use the legacy weighting by boundary_edge_length instead of by
+            boundary_edge_length^2 for backwards compatibility
```

Comment:
Include a `.. versionadded:: 0.45` directive for new keywords
```suggestion
        boundary_constraints: bool, default: False
            Use the legacy weighting by boundary_edge_length instead of by
            boundary_edge_length^2 for backwards compatibility
            
           .. versionadded:: 0.45
```